### PR TITLE
Fix incorrectly applied "removed" status for pings

### PR DESCRIPTION
**Pull request by wlach**
Instead, we'll just say that they're always in source until we can
determine otherwise (see https://github.com/mozilla/probe-scraper/issues/341)

### CODE REVIEW

**Comment by wlach** [comment]
This bug is very visible if you look at the pings page for Fenix:

https://dictionary.telemetry.mozilla.org/apps/fenix?itemType=pings&page=1

image

**Review by Iinh — APPROVED** [review]

**Comment by wlach** [comment]
Yuck, this actually only half-fixes the problem since app_ids suffer from the same thing. I think the right solution here is to only consider an item removed if the in_source annotation is *actually* false. Not all items have the "in source" annotation. Will update PR.

File: src/state/items.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export const isRemoved = (item) => {
+  // only consider an item expired if it has an in_source property
+  // and it is false
+  return item.in_source === false;
+};
+
+export const isExpired = (item) => {
```
Comment (wlach) [inline src/state/items.js:7]: It felt like it would be best if these two methods took the same argument.

**Review by Iinh — APPROVED** [review]
Thanks for your patience, this looks good!

**Pull request merged**